### ISSUE TITLE
Form plugin error

### ISSUE THREAD

**Issue by javichu666** [Flexget/Flexget#1845]
### Expected behaviour:
<login to rss feed (http://feeds2.feedburner.com/newpctorrent) in order to download torrents from url http://www.newpct.com/entrar/>

### Actual behaviour:
From 
### Steps to reproduce:
- Step 1: ...

#### Config:
```
    rss: http://feeds2.feedburner.com/newpctorrent
    form:
      url: http://www.newpct.com/entrar/
      userfield: userName
      passfield: userPass
      username: myuser
      password: mypassword
```
  
#### Log:
```
2017-05-26 20:22 CRITICAL task          series_sigo     BUG: Unhandled error in plugin form: expected name token at '<!<31?106:(132,0x0))'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 486, in __run_plugin
    return method(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/operate/formlogin.py", line 77, in on_task_start
    for form in br.forms():
  File "/usr/lib/python2.7/dist-packages/mechanize/_mechanize.py", line 420, in forms
    return self._factory.forms()
  File "/usr/lib/python2.7/dist-packages/mechanize/_html.py", line 557, in forms
    self._forms_factory.forms())
  File "/usr/lib/python2.7/dist-packages/mechanize/_html.py", line 237, in forms
    _urlunparse=_rfc3986.urlunsplit,
  File "/usr/lib/python2.7/dist-packages/mechanize/_form.py", line 844, in ParseResponseEx
    _urlunparse=_urlunparse,
  File "/usr/lib/python2.7/dist-packages/mechanize/_form.py", line 981, in _ParseFileEx
    fp.feed(data)
  File "/usr/lib/python2.7/dist-packages/mechanize/_form.py", line 760, in feed
    raise ParseError(exc)
ParseError: expected name token at '<!<31?106:(132,0x0))'
```

### Additional information:

- FlexGet version: 2.10.54 today and yesterday 2.10.40 it fails too
- Python version: Python 2.7.9
- Installation method: pip repository
- Using daemon: NO 
- OS and version:Linux raspberrypi 4.9.24-v7+ #993 SMP Wed Apr 26 18:01:23 BST 2017 armv7l GNU/Linux , Raspbian GNU/Linux 8 (jessie)
- Link to crash log:

**Comment by Aokromes** [comment]
test https://github.com/Flexget/Flexget/pull/1838

**Comment by rsalas** [comment]
Fix #1838 don't solve the login problem.

**Comment by rsalas** [comment]
New Fix #1847 solve the login problem.

**Comment by ChrisN06** [comment]
Hello,

I have an issue with form login too. I'm unable to login to yggtorrent.com.

Conf:
```
    cfscraper: yes
    form:
      url: https://yggtorrent.com/user/login
      userfield: id
      passfield: pass
      username: '{? ygg.usr ?}'
      password: '{? ygg.pwd ?}'
    html:
      url: "https://yggtorrent.com/torrents/filmvideo/2183-film?per_page=10&order=desc&sort=publish_date&page=0"
      title_from: link
```

LOG:
```
2017-10-22 11:09 DEBUG formlogin yggtorrent_1080p Exception posting login form.
Traceback (most recent call last):
File "/opt/rh/python27/root/usr/lib64/python2.7/site-packages/flexget/plugins/operate/formlogin.py", line 66, in on_task_start
br.open(url)
File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/mechanize/_mechanize.py", line 254, in open
return self.mechopen(url_or_request, data, timeout=timeout)
File "/opt/rh/python27/root/usr/lib/python2.7/site-packages/mechanize/_mechanize.py", line 310, in mechopen
raise response
httperror_seek_wrapper: HTTP Error 403: Forbidden
2017-10-22 11:09 CRITICAL formlogin yggtorrent_1080p Unable to post login form
2017-10-22 11:09 WARNING task yggtorrent_1080p Aborting task (plugin: form)
2017-10-22 11:09 DEBUG task_queue task yggtorrent_1080p aborted: TaskAbort(reason=Unable to post login form, silent=False)
```

```
$ flexget -V
2.10.102
You are on the latest release.
```

Thanks

**Comment by cvium** [comment]
Looks like the fix in #1847 is no longer relevant as per https://github.com/python-mechanize/mechanize/blob/master/ChangeLog#L50
